packages/react/src/Badge/Badge.tsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { forwardRef } from 'react'
+import type { ForwardedRef, HTMLAttributes, PropsWithChildren } from 'react'
+
+export interface BadgeProps extends PropsWithChildren<HTMLAttributes<HTMLElement>> {
+  color?: 'blue' | 'dark-blue' | 'green' | 'dark-green' | 'orange' | 'magenta' | 'purple' | 'yellow'
+}
+
+export const Badge = forwardRef(
+  ({ children, className, color = 'dark-green', ...restProps }: BadgeProps, ref: ForwardedRef<HTMLElement>) => (
+    <span {...restProps} ref={ref} className={clsx('amsterdam-badge', `amsterdam-badge--${color}`, className)}>
+      {children}
```

Comment:
I’m leaning towards using a `label` prop for the text. We want one or two words in here, maybe an icon someday. Rendering any children may open up too many options for misuse.

Reply:
Your suggestion of using `label: string | number` would also allow them to enter any string they want right? It would prevent abuse for something like a react node in a coloured block but you can do that with `SpotLight`.

Reply:
Yes, obviously, but a `label` prop does prevent people from putting paragraphs, lists, buttons, tables, images etc. in a `Badge` – like `children` would allow to.

Reply:
Changed it.